examples/ui/anchor_layout.rs

```diff
@@ -0,0 +1,164 @@
+//! Shows an "anchor layout" style of ui layout
+use bevy::prelude::*;
+
+const MARGIN: Val = Val::Px(12.);
+
+fn main() {
+    App::new()
+        .add_plugins(DefaultPlugins.set(WindowPlugin {
+            primary_window: Some(Window {
+                title: "Bevy Anchor Layout Example".to_string(),
+                ..default()
+            }),
+            ..default()
+        }))
+        .add_systems(Startup, spawn_layout)
+        .run();
+}
+
+fn spawn_layout(mut commands: Commands, asset_server: Res<AssetServer>) {
+    let font = asset_server.load("fonts/FiraSans-Bold.ttf");
+    commands.spawn(Camera2d);
+    commands
+        .spawn((
+            Node {
+                // fill the entire window
+                width: percent(100),
+                height: percent(100),
+                flex_direction: FlexDirection::Column,
+                align_items: AlignItems::Center,
+                padding: UiRect::all(MARGIN),
+                row_gap: MARGIN,
+                ..default()
+            },
+            BackgroundColor(Color::BLACK),
+        ))
+        .with_children(|builder| {
+            let rows = [
+                [
+                    (
+                        "left/top",
```

Comment:
These could show the actual layout constraints instead, like:
```
                        "Node {\n\tleft: px(10),\n\ttop: px(10),\n\t..default(),\n}",
```
if they can be made to fit.